src/jobs/integration-test.yaml

```diff
@@ -0,0 +1,43 @@
+description: |
+  Runs an integration test by creating a KIND cluster and executing it as a Go
+  test.
+
+  Uses the machine executor and replaces the Go version with a version
+  controlled by us since the preinstalled Go is very outdated.
+
+  Tests can be customized by providing a KIND config file or a setup script
+  which is run before the test is executed.
+environment:
+  GOPATH: /go
```

Comment:
Why do we need to change the GOPATH here? It's already set up there https://github.com/giantswarm/architect-orb/pull/81/files#diff-af2873c4ca693176b903875216a77aa9R22-R25

Reply:
I think its a leftover from previous PRs. I'll remove.